docs/installation.rst

```diff
@@ -51,7 +51,8 @@ Installing from npm
 
 You can install CasperJS using `npm <http://npmjs.org/>`_::
 
-    $ npm install -g casperjs
```

Comment:
Please re-enable this line in your PR, because we want to address this issue with casperjs@1.1.0-beta4.